src/reducers/tests/breakpoints.spec.js

```diff
@@ -11,20 +11,24 @@ import {
   getBreakpointsForSource,
   initialBreakpointsState
 } from "../breakpoints";
-import * as I from "immutable";
+
+import { createBreakpoint } from "../../utils/breakpoint";
 
 function initializeStateWith(data) {
-  const mappedData = I.Map(data);
   const state = initialBreakpointsState();
-  return state.set("breakpoints", mappedData);
+  state.breakpoints = data;
+  return state;
 }
 
 describe("Breakpoints Selectors", () => {
   it("it gets a breakpoint for an original source", () => {
     const sourceId = "server1.conn1.child1/source1/originalSource";
-    const matchingBreakpoints = { id1: { location: { sourceId } } };
+    const matchingBreakpoints = {
+      id1: createBreakpoint({ line: 1, sourceId: sourceId })
+    };
+
     const otherBreakpoints = {
-      id2: { location: { sourceId: "not-this-source" } }
+      id2: createBreakpoint({ line: 1, sourceId: "not-this-source" })
```

Comment:
this is a fun trick to keep the fixture data lean. We do the same thing for sources in some other tests

Reply:
Cool, I will update the tests with that